SilMock/tests/Google/Http/BatchTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Http;
```

Comment:
```suggestion
declare(strict_types=1);

namespace Http;
```

Reply:
I don't think so, since I am purposefully abusing the batch result type on the add to prove that it works good enough.